exotica_core/include/exotica_core/kinematic_tree.h

```diff
@@ -145,9 +154,16 @@ class KinematicTree : public Uncopyable
     const Eigen::MatrixXd& GetJointLimits() const { return joint_limits_; }
     void SetJointLimitsLower(Eigen::VectorXdRefConst lower_in);
     void SetJointLimitsUpper(Eigen::VectorXdRefConst upper_in);
+    void SetJointVelocityLimits(Eigen::VectorXdRefConst velocity_in);
+    void SetJointAccelerationLimits(Eigen::VectorXdRefConst acceleration_in);
+    void SetFloatingBaseLimitsPosXYZEulerZYX(const std::vector<double>& lower, const std::vector<double>& upper, const std::vector<double>& velocity, const std::vector<double>& acceleration);
```

Comment:
Velocity and accleration should be passed as a const double, not a vector

Reply:
Ah yes, I forgot to change this when I changed from vector to double